index.html

```diff
@@ -185,42 +184,48 @@ <h3>Skills</h3>
                     </ul>
                 </div>
             </section>
-        </div>
-
-        <div class="wrapper-menu" id="wrapper-menu">
-            <header class="header">
-                <nav class="nav">
-                    <img src="img/image_geometry_menu_1.svg" alt="">
-                    <a href="#" onclick="closeMenu()"><img class="menu-button" src="img/Union.svg" alt="Menu Buttom"></a>
-                </nav>
-            </header>
-
-            <section class="menu">
-                <div class="content-menu">
-                    <ul class="menu-list">
-                        <li>Hello</li>
-                        <li>Portfolio</li>
-                        <li>About</li>
-                        <li>Contact</li>
-                    </ul>
-                </div>
-                <div class="content-bottom">
-                    <img class="f-img" src="img/image_geometry_menu_2.svg" alt="">
+            
+            <section class="contact">
+                <div class="contact-form">
+                    <div class="title set">
+                        <h2>Get started</h2>
+                    </div>
+                    <div class="content set">
+                        <p class="description">
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.
+                        </p>
+                        <form class="user-form" action="https://formspree.io/f/xoqrloon" method="post">
+                            <label for="name">
+                                Your name
+                                <input class="name-font" type="text" name="name" id="name" maxlength="30" value="Anastasiia">
```

Comment:
- [ ] Kindly make sure this `input` has the `required` attribute as stated in the project requirements
- [ ] Please kindly include a `placeholder` attribute in place of the `value` attribute